build-aux/snap/snapcraft.yaml

```diff
@@ -305,6 +302,7 @@ parts:
             esac
             ;;
         esac
+        TAGS+=(snap osusergo)
```

Comment:
github.com/godbus/dbus seems to call user.Current() a couple of times in some fallback paths:

```
vendor/github.com/godbus/dbus/conn_other.go
80:     if currentUser, err := user.Current(); err != nil {

vendor/github.com/godbus/dbus/homedir_dynamic.go
10:     u, err := user.Current()
```

which could be incorrect with `osusergo`. Perhaps we should still pack libnss-extrausers to handle this edge case but make sure our code is using the osutil wrapper?

Reply:
Those cases should work. One is for the home dir, and if HOME defined it should work. Which should be 99.99% cases.

The other case is to get the uid, which should work correct in osusergo. It is just os.Getuid